src/main/scala/sparkz/core/api/http/PeersApiRoute.scala

```diff
@@ -187,16 +188,14 @@ case class PeersApiRoute(peerManager: ActorRef,
 
   def removeFromBlacklist: Route = (path("blacklist") & delete & withBasicAuth) {
     _ => {
-      entity(as[Json]) { json =>
-        val maybeAddress = json.asString.flatMap(addressAndPortRegexp.findFirstMatchIn)
-
-        maybeAddress match {
-          case None => ApiError(StatusCodes.BadRequest, s"address $maybeAddress is not well formatted")
+      entity(as[AddressBodyRequest]) { bodyRequest =>
```

Comment:
Shouldn't also the other APIs support IPv6 or just this one? Because e.g. removePeer still uses addressAndPortRegexp.

Reply:
agreed to postpone other ipv6 support